### ISSUE TITLE
Setup commands that run in a different container

### ISSUE THREAD

**Issue by charleskorn** [charleskorn/batect#286]
Continuation of discussion in #105, for the more advanced use case:

> some way to run commands in another container after another container becomes healthy but before any containers that depend on it start

**Comment by paulvalla-zorro** [comment]
That would be handy, we are trying to initialize a dynamodb running in one container, using an aws-cli container to create a table and once everything is ready, run integration tests in a third container.

**Comment by kokrange** [comment]
I think it's a pretty common scenario especially for database migration because most db containers do not have the environment to migrate itself. It would be nice to have some kind of short-lived container that do something only for the long-lived container and quit after finished, like k8s Jobs.
